src/main/controllers/ReasonableAdjustmentsController.ts

```diff
@@ -16,7 +17,7 @@ export default class ReasonableAdjustmentsController {
 
   private readonly reasonableAdjustmentsContent: FormContent = {
     fields: {
-      reasonableAdjustments: {
+      reasonable_adjustments: {
```

Comment:
Typically the fields on the frontend side are camel cased rather than snake cased, it helps distinguish the two

Reply:
Renamed the snake case fields to camel case